client/src/components/MovieCard/MovieCard.component.jsx

```diff
@@ -43,27 +60,72 @@ export const MovieCard = ({ src, title, genresList, id, year }) => {
     ["svg", "DIV", "BUTTON"].includes(event.target.nodeName) &&
     event.preventDefault();
 
+  const deleteMovie = () => deleteMovieMutation(id);
+
+  const editingErrorHandler = (error) => {
+    console.log(error);
+    setIsEditingError("Sth went wrong.");
+  };
+
+  const editingSuccessHandler = (title) =>
+    setIsEditingSuccess(`${title} was successfully edited.`);
+
+  let modalBodyContent, modalHeadingContent;
+
+  if (!isEditingSuccess && !isEditingError) {
+    if (optionValue.value === "edit") {
+      modalHeadingContent = "Edit";
+      modalBodyContent = (
+        <BaseForm
+          onEditingError={editingErrorHandler}
+          onEditingSuccess={editingSuccessHandler}
+          movieToEdit={{
+            title,
+            id,
+            posterPath,
+            releaseDate,
+            overview,
+            runtime,
+            voteAverage,
+            genres: genres.map((genre) => ({
```

Comment:
if you need to throw all props to another component
it better not to distruct props in args, better create const {a,b,c} = props;
and here you can have shorter list of props